scripts/_common.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+#=================================================
+# PERSONAL HELPERS
+#=================================================
+
+# Delete a file checksum from the app settings
+#
+# $app should be defined when calling this helper
+#
+# usage: ynh_remove_file_checksum file
+# | arg: file - The file for which the checksum will be deleted
+ynh_delete_file_checksum () {
+	local checksum_setting_name=checksum_${1//[\/ ]/_}	# Replace all '/' and ' ' by '_'
+	ynh_app_setting_delete $app $checksum_setting_name
+}
```

Comment:
Your requirement is at ynh 3.4, this helper is therefor not needed here.

Reply:
removed :)